src/lib/webcontainer.ts

```diff
@@ -267,6 +276,50 @@ async function launch_jsh() {
 	}
 }
 
+function parse_svelte_check(chunk: string) {
+	const result = chunk.trim().match(/^\d+\s(?<diagnostic>.*)$/);
```

Comment:
id love a comment for what this regex does 😄

Reply:
Lol I'll make sure to comment each one...I'm also writing another regex to check the version. Btw this is still a draft pr since I need to do other things